selfdrive/ui/update_translations.py

```diff
@@ -9,8 +9,21 @@
 TRANSLATIONS_DIR = os.path.join(UI_DIR, "translations")
 LANGUAGES_FILE = os.path.join(TRANSLATIONS_DIR, "languages.json")
 
+def generate_translations_include():
+  # offroad alerts
+  content = "// THIS IS AN AUTOGENERATED FILE, PLEASE EDIT alerts_offroad.json\n"
+  with open(os.path.join(BASEDIR, "selfdrive/controls/lib/alerts_offroad.json")) as f:
+    for alert in json.load(f).values():
+      content += "QT_TRANSLATE_NOOP(\"OffroadAlert\", \"" + alert['text'].replace('\n', '\\n').replace('\"', '\\"') + "\");\n"
```

Comment:
think it's fine to use raw strings to avoid the replaces, but up to you. I'd use f-strings here to build the lines